replication/gatys_et_al_2017/main.py

```diff
@@ -30,8 +30,11 @@ def figure_2_d(content_image, style_image):
             style_image,
             impl_params=args.impl_params,
         )
+        filename = make_output_filename(
+            "gatys_et_al_2017", "fig_2", "d", impl_params=args.impl_params
```

Comment:
With the extra comma at the end, `black` explodes this, which makes it easier to read.

```suggestion
            "gatys_et_al_2017", 
            "fig_2", 
            "d", 
            impl_params=args.impl_params,
```

Please apply this to all occurrences below.